--- gpsd-2.95/configure.ac~	2010-07-08 20:16:16.848861269 +0200
+++ gpsd-2.95/configure.ac	2010-07-08 20:17:38.965519312 +0200
@@ -227,13 +227,13 @@
 	   [Define if you have the external 'daylight' variable.])
 fi
 
-AC_CHECK_LIB(nsl, gethostbyname, LIBNSL="-lnsl")
+AC_CHECK_FUNC([gethostbyname], , [AC_CHECK_LIB(nsl, gethostbyname, LIBNSL="-lnsl")])
 AC_SUBST(LIBNSL)
-AC_CHECK_LIB(socket, socket, LIBSOCKET="-lsocket")
+AC_CHECK_FUNC([socket], , [AC_CHECK_LIB(socket, socket, LIBSOCKET="-lsocket")])
 AC_SUBST(LIBSOCKET)
 AC_CHECK_LIB(m, rint, LIBM="-lm")
 AC_SUBST(LIBM)
-AC_CHECK_LIB(c, open, LIBC="-lc")
+AC_CHECK_FUNC([open], , [AC_CHECK_LIB(c, open, LIBC="-lc")])
 AC_SUBST(LIBC)
 AC_CHECK_LIB(pthread, pthread_setcancelstate,
 	[LIBPTHREAD="-lpthread"
@@ -352,7 +352,7 @@
 dnl libncurses.  We define our own variables to reduce the risk of
 dnl breaking if autoconf changes (private) variable names.
 AC_CHECK_HEADERS(ncurses.h)
-AC_CHECK_LIB(ncurses, initscr, NCURSES_LIBS="-lncurses")
+AC_CHECK_LIB(ncurses, initscr, NCURSES_LIBS="-lncurses -ltinfo")
 AC_SUBST(NCURSES_LIBS)
 if test x"$NCURSES_LIBS" = x"" ; then
   AC_MSG_WARN([Not including curses support])
--- gpsd-2.95/Makefile.am.orig	2011-01-15 19:48:42.598366646 +0100
+++ gpsd-2.95/Makefile.am	2011-01-16 07:57:57.110356589 +0100
@@ -260,7 +260,7 @@
 
 nodist_libgpsd_la_SOURCES = packet_names.h ais_json.i
 libgps_la_LIBADD = $(LIBM) $(LIBC) $(LIBNSL) $(LIBSOCKET) $(LIBPTHREAD)
-libgpsd_la_LIBADD = $(LIBM) $(LIBC) $(LIBNSL) $(LIBSOCKET) $(LIBPTHREAD) $(BLUEZ_LIBS) libgps.la
+libgpsd_la_LIBADD = $(LIBM) $(LIBC) $(LIBNSL) $(LIBSOCKET) $(LIBPTHREAD) $(BLUEZ_LIBS) $(LIBUSB_LIBS) libgps.la
 
 noinst_SCRIPTS =
 
