--- gpsd-3.19/SConstruct.orig	2019-07-02 04:00:17.000000000 +0200
+++ gpsd-3.19/SConstruct	2019-10-04 17:36:33.068267039 +0200
@@ -2663,7 +2663,7 @@
 
 if env['systemd'] and not env["sysroot"]:
     systemctl_daemon_reload = Utility('systemctl-daemon-reload', '',
-                                      ['systemctl daemon-reload || true'])
+                                      ['true'])
     env.AlwaysBuild(systemctl_daemon_reload)
     env.Precious(systemctl_daemon_reload)
     env.Requires(systemctl_daemon_reload, systemd_install)
