--- gpsd-3.1/gpsd.h-tail.orig	2011-08-26 13:56:08.000000000 +0200
+++ gpsd-3.1/gpsd.h-tail	2011-09-27 20:47:19.111181574 +0200
@@ -758,7 +758,12 @@
 #define DEBUG_JSON	5	/* minimum level for verbose JSON debugging */
 # define libgps_debug_trace(args) (void) libgps_trace args
 extern int libgps_debuglevel;
-extern void libgps_dump_state(struct gps_data_t *);
+#ifdef __cplusplus
+extern "C"
+#else
+extern
+#endif
+void libgps_dump_state(struct gps_data_t *);
 #else
 # define libgps_debug_trace(args) /*@i1@*/do { } while (0)
 #endif /* LIBGPS_DEBUG */
