--- gpsd-3.8/SConstruct.orig	2013-03-01 18:51:51.019185370 +0100
+++ gpsd-3.8/SConstruct	2013-03-01 19:07:33.045812503 +0100
@@ -287,8 +287,6 @@
 # Honor the specified installation prefix in link paths.
 if env["sysroot"]:
     env.Prepend(LIBPATH=[env["sysroot"] + installdir('libdir', add_destdir=False)])
-if env["shared"]:
-    env.Prepend(RPATH=[installdir('libdir')])
 
 # Give deheader a way to set compiler flags
 if 'MORECFLAGS' in os.environ:
@@ -842,7 +840,8 @@
                            target="gpsd",
                            sources=libgpsd_sources,
                            version=libgpsd_version,
-                           parse_flags=usblibs + rtlibs + bluezlibs)
+                           parse_flags=usblibs + rtlibs + bluezlibs + ["-L.", "-lgps", "-lm"])
+env.Depends(compiled_gpsdlib, compiled_gpslib)
 
 libraries = [compiled_gpslib, compiled_gpsdlib]
 
@@ -976,7 +976,7 @@
 #          linking
 # The final executable will build but not be portable.
 
-env.StaticLibrary(target = 'libgps.a', source = libgps_sources)
+env.StaticLibrary(target = 'libgps.a', source = libgps_sources, parse_flags = dbus_libs)
 
 # Source groups
 
