--- gpsd-3.10/SConstruct~	2016-05-06 09:30:20.000000000 +0200
+++ gpsd-3.10/SConstruct	2016-05-06 09:46:02.440157339 +0200
@@ -1348,7 +1348,7 @@
 if not env['python']:
     python_install = []
 else:
-    python_lib_dir = sysconfig.get_python_lib(plat_specific=1)
+    python_lib_dir = sysconfig.get_python_lib(plat_specific=1,prefix=env['prefix'])
     python_module_dir = python_lib_dir + os.sep + 'gps'
     python_extensions_install = python_env.Install( DESTDIR + python_module_dir,
                                                     python_built_extensions)
